### ISSUE TITLE
Errors during build

### ISSUE THREAD

**Issue by kirugan** [xenia-project/xenia#1162]
I'm trying to build xenia on my PC (win10 x64) during build I get such errors:
```
LINK : warning LNK4001: no object files specified; libraries used [...\xenia\build\xenia-ui-window-vulkan-demo.vcxproj]
LINK : error LNK1218: warning treated as error; no output file generated [...\xenia\build\xenia-ui-window-vulkan-demo.vcxproj]
LINK : warning LNK4001: no object files specified; libraries used [...\xenia\build\xenia-hid-demo.vcxproj]
LINK : error LNK1218: warning treated as error; no output file generated [...\xenia\build\xenia-hid-demo.vcxproj]```

**Comment by gibbed** [comment]
It's something we need to fix, but you can ignore these errors as they just prevent the xenua-ui-vulkan-demo and xenia-hid-demo from being output.

**Comment by Triang3l** [comment]
On an unrelated note, Linux CI build currently fails due to https://github.com/xenia-project/xenia/blob/f450ae6a766316dc98e599b4a1bfb4543cbd90de/src/xenia/kernel/xam/xam_user.cc#L120 (conditional expression is ambiguous; 'int' can be converted to 'xe::be<unsigned int>' and vice versa). Should be fine with if-else rather than ?:.

**Comment by Parovozik** [comment]
Earlier, everything was build compiled without problems. On new builds I do everything just as earlier but because of these errors everything stops 
![bandicam 2018-05-29 02-50-32-303](https://user-images.githubusercontent.com/19775644/40632169-b11cae0e-62ec-11e8-82f0-84a807160115.jpg)

**Comment by gibbed** [comment]
@Parovozik Are you building with Visual Studio 2017?

**Comment by Parovozik** [comment]
@gibbed I kind of understood what my problem is. i be will reinstall the new version VS. Thank you!

**Comment by DrChat** [comment]
Fixed as of fb9eb8489381af1340385be3dfd1eee9d0a0a58d.
